### PR TITLE
Calvin's Pull request 

### PR DESCRIPTION
**Pull request by 812967**
Predator 2 
- four snake arms reach out and grab prey within range of 100 pixels and then returns to creature 
- changes color over time 
- dies over time 
- eats predator 3 

Still needs to: 
- reproduce 
- snake arms can't reach prey more than once (eats a total of 4 times in its life)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -11,6 +12,17 @@ function init(){
 
 function animate(){
   world.run();
+  for(let i = 0; i<world.creatures.pred2.length; i++){
```
Comment (810218) [inline main.js]: This should be in world.js

**Review by 810218 — CHANGES_REQUESTED** [review]
He needs to change the running of his predator from main.js to world.js

**Comment by 812967** [comment]
Predator 2
- four snake arms reach out and grab prey within range of 100 pixels and then returns to creature
- changes color over time
- dies over time
- eats predator 3

Still needs to:
- reproduce
- snake arms can't reach prey more than once (eats a total of 4 times in its life)

**Comment by sanjanksen** [comment]
Change everything. Quite spleindiferiousigenousiriditory

**Comment by 810636** [comment]
slay

**Pull request merged**